components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1056,7 +1063,14 @@ public MetadataStore getMetadataStoreForConversion() {
     int realSeries = getSeries();
     for (int i=0; i<getSeriesCount(); i++) {
       setSeries(i);
```

Comment:
I think this entire function belongs at the end of `initFile`, and should be removed entirely once moved.  Why aren't we setting these values along with the rest of the metadata?  I think this is historical cruft which could be dropped.